.github/workflows/tox.yml

```
@@ -179,6 +179,11 @@ jobs:
       with:
         name: logs.zip
         path: .tox/**/log/
+    - name: Report junit failures
+      uses: shyim/junit-report-annotations-action@master
+      with:
+        stripFromPath: .tox/
```

`path` is still necessary. otherwise `*.xml` won't match our reports
```suggestion
        path: .tox/TEST-*.xml
        stripFromPath: .tox/
```

---

```suggestion
```